content-repo/extra-docs/releases/20.5.2.md

```diff
@@ -0,0 +1,271 @@
+  - Replaced the spaces in the URL context output of the **!panorama-create-edl** command to *%20*.
+- __Generic SQL__  
+Fixed an issue where connecting to an MS SQL database using an encrypted connection failed.
+- __Tanium__  
+Fixed an issue where the output of some results was malformed.
+- __VirusTotal - Private API__  
+Fixed an issue where running file related commands would raise an error.
+- __Tanium Threat Response__  
+Fixed an issue where ***tanium-tr-get-downloaded-file*** retrieved a malformed file.
+- __URLhaus__  
+  - Added the *Number of retries* parameter which determines how many times a command should attempt before raising an error.
+  - Fixed an issue where the ***urlhaus-download-sample*** command would raise an error in cases where results were found.
+- __RTIR__  
+  - Fixed an issue where the ***fetch-incidents*** and ***search-tickets*** commands did not behave as expected.
+  - Fixed an issue where the test module did not work as expected.
+  - Added the *Fetch limit* parameter to the instance configuration, which specified the maximum number of results to fetch.
+  - Added the *results_limit* argument to the ***search-tickets*** command, which specified the maximum number of results to return.
+- __SplunkPy__  
+Added support for HTTPS handler, which uses tbhe Python **requests** library.
+- __Palo Alto Networks PAN-OS EDL Service__  
+  - Removed the *panos_compatible* parameter. All indicators exported by this integrations will be PAN-OS compatible.
+  - Added request parameters that are passed in the URL.
+- __OPSWAT-Metadefender v2__  
+Fixed an issue where running file related commands would raise an error.
+- __ServiceNow v2__  
+  - Added the *incident_name* parameter, to allow the user to choose the column from ServiceNow on which the fetched incidents will be named.
+  - Fixed an issue where system proxy settings were always being used.
+  - Fixed an issue where ***fetch-incidents*** command with attachments did not work as expected.
+- __RSA Archer__  
+  - Fixed an issue where several commands would not work as expected in case they were performed on app ID 411.
+  - Fixed an issue where type 4 fields were not displayed in ***archer-search-records*** command results.
+- __Microsoft Graph Mail Single User__  
+Fixed an issue where some emails were not fetched as incidents.
+- __Expanse__  
+- Added support for filtering incident creation by Expanse Exposure severity level.
+- __MongoDB__  
+Fixed an issue when pulling an object that contains a date.
+- __Azure Compute v2__  
+Added support to authenticate using a self-deployed Azure application.
+- __Palo Alto Networks BPA__  
+  - Added an argument which enables you to download a Panorama report.
+  - Fixed an issue where proxy settings were not handled properly.
+
+#### Deprecated Integration
+- __ServiceNow__  
+Use the *ServiceNow v2* integration instead (available from 5.0.0).
```

Comment:
lets be consistent with the "from version" format
here we say `available from 5.0.0` and a few lines to the bottom we say `Available from Cortex XSOAR 5.5`